cryptography/build.gradle.kts

```diff
@@ -9,16 +9,17 @@ version = "0.0.1-SNAPSHOT"
 
 android {
     compileSdk = Android.Sdk.compile
-    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
     defaultConfig {
         minSdk = Android.Sdk.min
         targetSdk = Android.Sdk.target
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
     }
     compileOptions {
         sourceCompatibility = JavaVersion.VERSION_1_8
         targetCompatibility = JavaVersion.VERSION_1_8
     }
-
+    sourceSets["main"].manifest.srcFile("src/androidMain/AndroidManifest.xml")
+    sourceSets.remove(sourceSets["test"])
```

Comment:
Just for the future, maybe we can add a comment here and explain why unit tests are removed from Android.

Something like `// Android Unit tests removed, as it's currently impossible to run native-through-NDK code on simple Unit tests`